content/_index.md

```diff
@@ -12,11 +12,10 @@ description: |
 # NeoForge installer files
 You can find a direct link to our latest installer files below.
 
-{{<projects>}}
-
+<!--Change with the latest version here-->
+{{< files "20.4" >}}
```

Comment:
Do we want to hardcode this latest version here or could we maybe fetch it from the maven somehow?

Reply:
Will be very helpful; like the `projects` website. Will do some research...